src/material/select/src/select.spec.ts

```diff
@@ -9,8 +9,9 @@ import { FormGroup, ReactiveFormsModule } from '@angular/forms';
 import { FormlyModule, FormlyForm } from '@ngx-formly/core';
 import { FormlySelectModule } from '@ngx-formly/core/select';
 import { FormlyFieldSelect } from './select.type';
-import { of as observableOf } from 'rxjs';
+import { of as observableOf, of } from 'rxjs';
```

Comment:
`observableOf` = `of`

Reply:
oh yes, probably auto-imported by VSCode 👍